### ISSUE TITLE
No module named 'lantz.log'

### ISSUE THREAD

**Issue by jondoesntgit** [lantzproject/lantz#6]
After the install that I followed from the README.md, I get the following error:

```
>>> from lantz.log import log_to_screen
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'lantz.log'
>>>
```

**Comment by hgrecco** [comment]
This is fixed in the current version in github and will be on the next release.

**Comment by hgrecco** [comment]
This is fixed in 0.5.2
